lib/bamboo/mandrill_adapter.ex

```diff
@@ -0,0 +1,46 @@
+defmodule Bamboo.MandrillAdapter do
+  @send_message_path "api/1.0/messages/send.json"
+
+  def deliver(email, config) do
+    api_key = Keyword.fetch!(config, :api_key)
+    params = email |> convert_to_mandrill_params(api_key) |> Poison.encode!
+    HTTPoison.post("http://localhost:4001/#{@send_message_path}", params, headers)
```

Comment:
I'll extract this URI so it pulls from the ENV, but I'll do that in a separate PR. Don't want to make this one too big